src/main/java/mchorse/metamorph/api/morphs/AbstractMorph.java

```diff
@@ -138,7 +138,7 @@ public void morph(EntityLivingBase target)
     public void demorph(EntityLivingBase target)
     {
         /* 20 is default player's health */
-        this.setHealth(target, this.lastHealth < 20 ? 20 : this.lastHealth);
+        this.setHealth(target, this.lastHealth <= 0.0F ? 20.0F : this.lastHealth);
```

Comment:
I assume the idea was to check if lastHealth was defined. Now that we're working with the base health modifier proper, it's quite rare that lastHealth will be anything other than 20, but you never know.